test/test_partner_sites.py

```diff
@@ -67,13 +67,9 @@ def test_archive_status_retrieve_sequence(self):
             self.skipTest("This test works only against the sqlite db")
 
         host_name = getHostName()
-        domain_name = getDomain()
-        host_name_fqdn = host_name
-        if domain_name is not None:
-            host_name_fqdn = "{0}.{1}".format(host_name, domain_name)
         sample_file_name = "SmallFile.fits"
         sample_file_path = os.path.join("src", sample_file_name)
```

Comment:
This will need to be `sample_file_path = self.resource(os.path.join("src", sample_file_name))`; that will correctly calculate the path to the source file regardless of where you are running the tests from. Tests locally failed because src/SmallFile.fits cannot be found from the top-level directory of the repository, which is where tests are usually run from.